src/components/Modal/Modal.mdx

```diff
@@ -5,13 +5,18 @@ import { Primary } from './Modal.stories';
 
```

Comment:
```suggestion

Both header and icon can be supplied either as a prop or slot, but they should not be used at the same time because this will cause both to be rendered.
```

Reply:
Added this to the docs